File: 02. C# Part2/03. Methods-Homework/03. EnglishDigit/EnglishDigit.cs
using System;

    //Write a method that returns the last digit of given integer as an English word.

    class EnglishDigit
    {
        static string word = null;
        static int lastDigit = 0;
        static void Main()
        {
            Console.WriteLine("Please enter a number: ");
            int input = int.Parse(Console.ReadLine());
            LastDigit(LastDigit(input));
            EnglishWord(lastDigit);
        }

        private static void EnglishWord(int input)
        {
            switch (input)
            {
                case 0: word = "zero"; break;
                case 1: word = "one"; break;
                case 2: word = "two"; break;
                case 3: word = "three"; break;
                case 4: word = "four"; break;
                case 5: word = "five"; break;
                case 6: word = "six"; break;
                case 7: word = "seven"; break;
                case 8: word = "eight"; break;
                case 9: word = "nine"; break;
            }
            Console.WriteLine(word);
        }

        private static int LastDigit(int input)
        {
            return lastDigit = input % 10;
            
        }
    }

File: 06. JavaScript Basics/06. Functions-Homework/06. Functions-Homework/01. EnglishDigit/EnglishDigit.js
//Write a function that returns the last digit of given integer as an English word.

function findLastDigit(a) {
    jsConsole.write("The last digit as word from the number " + a + " is: "  );
    var digit = a % 10;
    switch (digit) {
        case 1:
            jsConsole.writeLine("one");
            break;
        case 2:
            jsConsole.writeLine("two");
            break;
        case 3:
            jsConsole.writeLine("three");
            break;
        case 4:
            jsConsole.writeLine("four");
            break;
        case 5:
            jsConsole.writeLine("five");
            break;
        case 6:
            jsConsole.writeLine("six");
            break;
        case 7:
            jsConsole.writeLine("seven");
            break;
        case 8:
            jsConsole.writeLine("eight");
            break;
        case 9:
            jsConsole.writeLine("nine");
            break;
        default:
            jsConsole.writeLine("zero")
    }
}
findLastDigit(512);
findLastDigit(1024);
findLastDigit(12309);